rosidl_typesupport_connext_c/resource/msg__type_support_c.cpp.template

```diff
@@ -182,7 +182,7 @@ convert_ros_to_dds(const void * untyped_ros_message, void * untyped_dds_message)
       return false;
     }
     DDS_Long length = static_cast<DDS_Long>(size);
-    dds_message->@(field.name)_.length(length);
+    dds_message->@(field.name)_.ensure_length(length, 2 * length);
```

Comment:
If the `dds_message` is temporary, then as @dirk-thomas suggested it should be `length, length`. @mikaelarguedas can you check to make sure this message is temporary and update if it is?